xla/service/gpu/gemm_rewriter.cc

```diff
@@ -2261,10 +2268,14 @@ class GemmWorkspaceRewriteVisitor : public DfsHloRewriteVisitor {
       TF_ASSIGN_OR_RETURN(
           has_aux_output,
           xla::gpu::gpublas_lt::EpilogueHasAuxiliaryOutput(epilogue));
+      if (config.has_has_damax_output()) {
```

Comment:
See above. I suggest making this a non-optional bool or renaming it to damax_output.